frigate/embeddings/maintainer.py

```diff
@@ -640,15 +640,16 @@ def _read_and_crop_snapshot(self, event: Event, camera_config) -> bytes | None:
             return None
 
     def handle_regenerate_description(self, event_id: str, source: str) -> None:
+        # We regenerate a description for an event unconditionally (whether genai is enabled or not for this camera)
+
         try:
             event: Event = Event.get(Event.id == event_id)
         except DoesNotExist:
             logger.error(f"Event {event_id} not found for description regeneration")
             return
 
-        camera_config = self.config.cameras[event.camera]
-        if not camera_config.genai.enabled or self.genai_client is None:
```

Comment:
Rather than another config option, I'd prefer to see the `force` param just passed through the publisher and then just checked here. For example:

`def handle_regenerate_description(self, event_id: str, source: str, force: bool = False) -> None:`

Reply:
Fixed